backend/grants/admin.py

```diff
@@ -411,3 +417,76 @@ def save_form(self, request, form, change):
 
     class Media:
         js = ["admin/js/jquery.init.js"]
+
+
+@admin.register(GrantRecap)
+class GrantsRecap(admin.ModelAdmin):
+    list_filter = ("conference",)
+    qs = None
+
+    def has_change_permission(self, *args, **kwargs) -> bool:
+        return False
+
+    def get_queryset(self, request):
+        if self.qs:
+            return self.qs
+
+        self.qs = super().get_queryset(request)
+        filters = dict(request.GET.items())
+
+        if filters:
+            self.qs = self.qs.filter(**filters)
+
+        return self.qs
+
+    def changelist_view(self, request, extra_context=None):
+        qs = self.get_queryset(request).order_by("traveling_from")
+
+        results = []
+        for country_code, group in groupby(list(qs), key=lambda k: k.traveling_from):
+            country = countries.get(code=country_code)
+            if not country:
+                continue
+            grants = list(group)
+            counter = Counter([g.status for g in grants])
+            results.append(
+                {
+                    "continent": country.continent.name,
+                    "country": country.name,
+                    "total": int(sum([g.total_amount for g in grants])),
+                    "count": len(grants),
+                    "pending": counter.get(Grant.Status.pending, 0),
+                    "rejected": counter.get(Grant.Status.rejected, 0),
+                    "approved": counter.get(Grant.Status.approved, 0),
+                    "waiting_list": counter.get(Grant.Status.waiting_list, 0)
+                    + counter.get(Grant.Status.waiting_list_maybe, 0),
+                    "waiting_for_confirmation": counter.get(
+                        Grant.Status.waiting_for_confirmation, 0
+                    ),
+                    "refused": counter.get(Grant.Status.refused, 0),
+                    "confirmed": counter.get(Grant.Status.confirmed, 0),
+                }
+            )
+
+        results = sorted(results, key=lambda k: k["count"], reverse=True)
+        results = sorted(results, key=lambda k: (k["continent"],))
```

Comment:
wasn't it possible to do it in one call to `sorted`?

 ```suggestion
        results = sorted(results, key=lambda k: (-k["count"], k["continent"]))
```

something like this? I might be wrong, I didn't try it